.env.template

```diff
@@ -0,0 +1 @@
+REACT_APP_DB_CONNECT=
```

Comment:
Not full template

Reply:
Each developer must enter the data to run locally.